### PR TITLE
Remove Page macro DataTransfer

### PR DESCRIPTION
**Pull request by hamishwillee**
Removes the `Page` macro from DragTransfer sub topics as part of fixing #3196

This is to explore what to do when Page macro is used to include **See also** sections into methods, properties etc from the top level (as queried in https://github.com/mdn/content/pull/3008#issuecomment-804606895). In this case some, but not all, elements like  [DataTransfer.addElement](https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/addElement) included the section from [DataTransfer](https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/addElement). 

What I have done is deleted the inclusion. My thinking is that there is already a link to the parent where the see also information is provided. In method doc I don't expect a see also, and it isn't worth maintaining one that can get out of sync with the parent. 

If this approach is OK I will extend it to the similar examples.

### CODE REVIEW

File: files/en-us/web/api/dragevent/index.html [hunk 1]
```diff
@@ -103,4 +103,10 @@ <h2 id="Browser_compatibility">Browser compatibility</h2>
 
 <h2 id="See_also">See also</h2>
 
-<p>{{page("/en-US/docs/Web/API/DataTransfer", "See also")}}</p>
+<ul>
```
Comment (hamishwillee) [inline files/en-us/web/api/dragevent/index.html:106]: This is the exception - it is a top level item that was including the see also. For this case I have duplicated the content.

**Comment by hamishwillee** [comment]
@wbamberg or @chrisdavidmills When you get a chance can you please look at this? If you're happy with the approach I can apply to the other similar instances.

**Review by chrisdavidmills — COMMENTED** [review]
This seems reasonable to me, although usually we like to have a "See also" section if possible.

@wbamberg in the recipe structures you've put together so far, is See also optional, or mandatory?

**Comment by hamishwillee** [comment]
> In this case, is it really the case that there's no useful "see also"? e.g. could https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/effectAllowed not include links to at least dragstart?

- It is always _possible_ to add a see also but I'm not sure it makes sense to do so. Using `dragstart` as an example - it is already referenced in the second paragraph of the introduction - so if you were going to see it, you already would have done so.
- It is not that there isn't anything useful we could add here - the set I removed because they use the page macro make sense because every method/property of a thing shares the same kinds of related material. However I chose not to manually duplicate them because then we have so much duplication and potential for things getting out of sync. 
- In other words, we're talking a trade off

Could this be automated? I.e. if no seealso section is defined in a property/sub page the see also of the first parent that has one is inserted by the toolchain?

If not this is kind of down to making a decision on accepting duplication or putting in some workaround.

**Comment by hamishwillee** [comment]
@chrisdavidmills @wbamberg Could I have a "ruling" on how you want to handle this, as per my comment above? In summary - options:
- Agree a SeeAlso that duplicates the parent is not needed.
- If it is then:
  - Recreate parent see also manually
  - Recreate keeping the page macro
  - Update yari to do this automatically if no SeeAlso is defined.
  - Make up a page specific SeeAlso whether or not it is actually useful.

**Comment by chrisdavidmills** [comment]
>     * Recreate parent see also manually

I'd do this for now. Our devs are so busy that we are unlikely to get their time to implement an automated solution at the moment.

**Comment by Ryuno-Ki** [comment]
>     * so if you were going to see it, you already would have done so

From personal experience, I can tell you that this would not be the case.
I'm skimming the first sentence, boxes (spec, possible values etc) and the See Also before I decide to read.

**Comment by hamishwillee** [comment]
@wbamberg @chrisdavidmills Ready for review (PageMacro is replaced with the equivalent text).

**Comment by hamishwillee** [comment]
Ping. Still ready for review/merging.

**Review by chrisdavidmills — APPROVED** [review]
See alsos implemented, and ready to go...cool, thanks @hamishwillee !
